silo/tests/test.py

```diff
@@ -21,6 +22,42 @@
 from tola.util import (addColsToSilo, hideSiloColumns, getColToTypeDict,
                        getSiloColumnNames)
 
+from celery.exceptions import Retry
+from mock import patch
+from silo.tasks import async_rand, process_silo
+
+
+class CeleryTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.factory = RequestFactory()
+        self.user = User.objects.create_user(username="bob", email="bob@email.com", password="tola123")
+        self.silo = Silo.objects.create(name="Test Silo", owner=self.user,
+                                   public=False, create_date=timezone.now())
+
+        self.read_type = ReadType.objects.create(read_type="CSV")
+        upload_file = open('test.csv', 'rb')
+        self.read = Read.objects.create(owner=self.user, type=self.read_type,
```

Comment:
Use the factories to create data. If the model that you wanna use isn't included, just create a factory for it.